traits/src/lib.rs

```diff
@@ -12,6 +12,9 @@ pub trait MultiCurrency<AccountId> {
 	/// The balance of an account.
 	type Balance: SimpleArithmetic + FullCodec + Copy + MaybeSerializeDeserialize + Debug + Default;
 
+	/// The error type.
+	type Error;
```

Comment:
```suggestion
	type Error: Into<&'static str>;
```

For better compatibility